src/catalog/src/durable/debug.rs

```diff
@@ -0,0 +1,456 @@
+            CollectionType::ClusterReplica => "compute_replicas",
+            CollectionType::Comment => "comments",
+            CollectionType::Config => "config",
+            CollectionType::Database => "database",
+            CollectionType::DefaultPrivilege => "default_privileges",
+            CollectionType::IdAllocator => "id_alloc",
+            CollectionType::Item => "item",
+            CollectionType::Role => "role",
+            CollectionType::Schema => "schema",
+            CollectionType::Setting => "setting",
+            CollectionType::StorageUsage => "storage_usage",
+            CollectionType::SystemConfiguration => "system_configuration",
+            CollectionType::SystemItemMapping => "system_gid_mapping",
+            CollectionType::SystemPrivilege => "system_privileges",
+            CollectionType::Timestamp => "timestamp",
+        }
+    }
+}
+
+impl FromStr for CollectionType {
+    type Err = anyhow::Error;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        if s == CollectionType::AuditLog.name() {
+            Ok(CollectionType::AuditLog)
+        } else if s == CollectionType::Cluster.name() {
+            Ok(CollectionType::Cluster)
+        } else if s == CollectionType::ClusterIntrospectionSourceIndex.name() {
+            Ok(CollectionType::ClusterIntrospectionSourceIndex)
+        } else if s == CollectionType::ClusterReplica.name() {
+            Ok(CollectionType::ClusterReplica)
+        } else if s == CollectionType::Comment.name() {
+            Ok(CollectionType::Comment)
+        } else if s == CollectionType::Config.name() {
+            Ok(CollectionType::Config)
+        } else if s == CollectionType::Database.name() {
+            Ok(CollectionType::Database)
+        } else if s == CollectionType::DefaultPrivilege.name() {
+            Ok(CollectionType::DefaultPrivilege)
+        } else if s == CollectionType::IdAllocator.name() {
+            Ok(CollectionType::IdAllocator)
+        } else if s == CollectionType::Item.name() {
+            Ok(CollectionType::Item)
+        } else if s == CollectionType::Role.name() {
+            Ok(CollectionType::Role)
+        } else if s == CollectionType::Schema.name() {
+            Ok(CollectionType::Schema)
+        } else if s == CollectionType::Setting.name() {
+            Ok(CollectionType::Setting)
+        } else if s == CollectionType::StorageUsage.name() {
+            Ok(CollectionType::StorageUsage)
+        } else if s == CollectionType::SystemItemMapping.name() {
+            Ok(CollectionType::SystemItemMapping)
+        } else if s == CollectionType::SystemConfiguration.name() {
+            Ok(CollectionType::SystemConfiguration)
+        } else if s == CollectionType::Timestamp.name() {
+            Ok(CollectionType::Timestamp)
+        } else {
+            anyhow::bail!("unknown collection {s}")
+        }
+    }
+}
```

Comment:
I'm going to update this with https://docs.rs/serde_plain/latest/serde_plain/ to make it simpler.